README.md

```diff
@@ -74,22 +74,23 @@ great detail when it comes to deploying Tide on GCP.
 
 ### Cloning
 
-Ensure you're in the directory where you would like to install Tide:
+To ensure easy opperation, clode using Go.
 
 ```
-git clone -b develop --recursive https://github.com/xwp/go-tide.git tide
+go get github.com/xwp/go-tide
 ```
 
 Change to Tide working directory:  
 
 ```
-cd tide
+cd $GOPATH/src/github.com/xwp/go-tide
 ```
```

Comment:
followed by....  `git remote set-url origin git@github.com:xwp/go-tide` to make dev easier